tests/resources/contracts/account_mock.cairo

```diff
@@ -0,0 +1,8 @@
+%lang starknet
+
+# The StarkNet compiler checks for an external "__execute__" method
+# if found, it expects an "--account_contract" flag during compilation
```

Comment:
Is this true? I would have thought it's the other way around: only look for a `__execute__` method if the `--account_contract` flag is present.

Reply:
Yeah, it actually checks both ways